qa/integration-tests/src/test/java/io/camunda/it/auth/UnprotectedApiIT.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.auth;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+
+import io.camunda.client.CamundaClient;
+import io.camunda.it.utils.CamundaMultiDBExtension;
+import io.camunda.it.utils.MultiDbTest;
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+@MultiDbTest
```

Comment:
adapted to new MultiDbTest-setup introduced by the exporter team pairing with @ChrisKujawa.
He will look into migrating the existing auth tests to it next and follow-up.